salt/pythontest/init.sls

```diff
@@ -4,6 +4,25 @@ include:
 git:
   pkg.installed
 
+ftp:
```

Comment:
if we can change this, a `ftp_username` configuration in `vsftpd.conf` would be nice.